frontend/app/assets/javascripts/spree/frontend/views/spree/shared/variant_select.js

```diff
@@ -0,0 +1,87 @@
+let getQueryString = window.location.search
```

Comment:
Can you change all features from ES6 to ES5 ('var' instead of 'let' and 'const' for example)? We use sprockets here and this will cause problems with deploys.

Reply:
Updated those.